### PR TITLE
Add language directories in _data and wiki to .gitignore

### PR DESCRIPTION
**Pull request by softins**
**Short description of changes**

Add the non-English translation directories from `_data` and `wiki` into `.gitignore`

**Context: Fixes an issue? Related issues**

Once translations have been run on a dev system, all the non-English dirs in `_data` and `wiki` are flagged by `git status`.
This just ignores them to make the output tidier. It will also prevent them accidentally being added to a commit by `git add`.

**Status of this Pull Request**

Ready

**What is missing until this pull request can be merged?**

Nothing

**Does this need translation?**
No

## Checklist

- [x] I've verified that this Pull Request follows the [general code principles](https://github.com/jamulussoftware/jamulus/blob/main/CONTRIBUTING.md#jamulus-projectsource-code-general-principles)
- [x] I waited some time after this Pull Request was opened and all GitHub checks completed without errors. 
- [x] I'm sure that this Pull Request goes to the correct branch

### CODE REVIEW

**Comment by pljones** [comment]
Is this targeting 3.11.0?  If so, please add to the board.

**Comment by softins** [comment]
> Is this targeting 3.11.0? If so, please add to the board.

I assume you meant add the milestone. It may as well go into the web update for 3.11.0, as it's small, non-invasive and just makes working tidier.

**Comment by pljones** [comment]
I meant the project tracking board:
https://github.com/orgs/jamulussoftware/projects/5/views/5

**Comment by softins** [comment]
OK I think I've added it right. Haven't used the project board before.

File: .gitignore [hunk 1]
```diff
@@ -1,2 +1,7 @@
 .jekyll-cache
 _site
+_data/*/
+!_data/en/
```
Comment (ann0see) [inline .gitignore:4]: You may want to exclude the README also.

**Review by ann0see — CHANGES_REQUESTED** [review]

File: .gitignore (hunk 1 above)
Comment (softins) [reply .gitignore:4]: No need. The trailing slash on `_data/*/` only matches directory names to ignore, so plain files within `_data` are not ignored. That's the reason I included the trailing slash.

Comment (ann0see) [reply .gitignore:4]: Ok. Fair point.

**Review by ann0see — APPROVED** [review]

**Pull request merged**